protos/search3.proto

```diff
@@ -1,21 +1,32 @@
 // -*-java-*-
+
 syntax = "proto3";
-//option java_multiple_files = true;
 option java_package = "com.cloudant.search3.grpc";
+import "google/protobuf/empty.proto";
 
 service Search {
     // Index operations
-    rpc Delete(Index) returns (ServiceResponse);
+    rpc Delete(Index) returns (google.protobuf.Empty);
     rpc GetUpdateSequence(Index) returns (UpdateSeq);
     rpc Info(Index) returns (InfoResponse);
-    rpc SetUpdateSequence(SetUpdateSeq) returns (ServiceResponse);
+    rpc SetUpdateSequence(SetUpdateSeq) returns (google.protobuf.Empty);
 
     // Read operations
     rpc Search(SearchRequest) returns (SearchResponse);
     rpc GroupSearch(GroupSearchRequest) returns (GroupSearchResponse);
 
     // Write operations
-    rpc Update(stream DocumentUpdate) returns (ServiceResponse);    
+    rpc UpdateDocument(DocumentUpdate) returns (google.protobuf.Empty);
+    rpc DeleteDocument(DocumentDelete) returns (google.protobuf.Empty);
```

Comment:
this is out of date, these return StreamResponse now.

Reply:
search3.proto won't be part of the normal branch. we still need to figure out how to `wget it from search3-java` with versioning.